src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callsLimit = 3;
+
+  return function(a) {
+    return function(b) {
+      if (callsLimit <= 0 && callsLimit % 2 === 0) {
+        callsLimit--;
+
+        return 'Bzzz... Error!';
```

Comment:
here can use variable

Reply:
I agree with you, but it is confusing that it is used in the code only once. I want to know the opinion of mentors about this

Reply:
i think it's okay as it is